ansible/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 - hosts: "all"
+  vars_files:
+    - "vars.yml"
```

Comment:
If you use `host_vars` for this, Ansible will pick this file up [automatically](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#host-variables) and tie it to the specific inventory host.